doc/user_guide/user_guide.md

```diff
@@ -285,9 +285,19 @@ severely. Available task_types are the same as the names of our available UDFs,
 `filling_mask`, `question_answering`, `sequence_classification`, `text_generation`, `token_classification`,
 `translation` and`zero_shot_classification`.
 
-If you need to do ths task using a python function instead of ta shell command,
-we provide `exasol_transformers_extension.upload_model.upload_model_to_bfs_location()`. This function takes the same
-arguments, and additionally a bucketfs_location to upload the model to.
+#### Installation via a Python Function
+
+Alternatively you can install a Huggingface model using a python function instead of a shell command.
+
+Function
+`exasol_transformers_extension.utils.model_utils.install_huggingface_model()` expects the following arguments
+* A BucketFS location
+* Some of the arguments wrapped into a `BucketFSModelSpecification`
```

Comment:
can you add which arguments? or soe explanation about the BucketFSModelSpecification? i think it was not mentioned in the user guide previousl, because it was only used internally.